### PR TITLE
Closes #3 - Reroute to /Reminders

### LINKED ISSUE #3: Reroute to Reminders
When the user visits the root of the application, they should be redirected to `'/reminders'` and should see a list of all reminders on the page. This means making a hopefully small adjustment to your `router.js` file.  

**Hot Tip**: This will make your previous test fail (if you had it passing). Make the necessary changes in the test(s).

### PR DESCRIPTION
**Pull request by MilkMan90**
@martensonbj @tman22 @stevekinney @nfosterky

## Purpose

When the user visits the root of the application, they should be redirected to '/reminders' and should see a list of all reminders on the page. This means making a hopefully small adjustment to your router.js file.

Hot Tip: This will make your previous test fail (if you had it passing). Make the necessary changes in the test(s).

## Approach

Solves issue by rerouting to /reminders route when directing towards index route of '/'

### Learning

Referenced @martensonbj's [Ember Router Lesson](http://frontend.turing.io/lessons/ember-routing.html) to find syntax for redirecting. Then we used `ember g route index` to generate an index route, and implemented the redirecting code there.

### Open Questions and Pre-Merge TODOs

- [x] render reminder items to page
- [x] reroute to /reminders when directing towards root index
- [x] updated test to account for redirection
- [ ] get all tests passing by adding links for each reminder

### Test coverage 

Updated existing tests to account for change with redirection

### Merge Dependencies and Related Work

#1 

### Follow-up tasks

#4 See Individual Reminder

#### Before

screen shot 2016-12-06 at 3 48 21 pm

#### After

screen shot 2016-12-07 at 12 59 52 pm

### CODE REVIEW

File: app/templates/index.hbs [hunk 1]
```diff
@@ -0,0 +1 @@
+{{outlet}}
```
Comment (martensonbj) [inline app/templates/index.hbs]: If we're not using this file lets get rid of it.

File: app/models/reminder.js [hunk 2]
```diff
@@ -4,5 +4,5 @@ export default DS.Model.extend({
   title: DS.attr('string'),
   date: DS.attr('date'),
   notes: DS.attr('string'),
-  pinned: false
+  pinned: DS.attr('boolean', { defaultValue: false })
```
Comment (martensonbj) [inline app/models/reminder.js]: Great catch making this a default value 👍

File: tests/unit/routes/index-test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+moduleFor('route:index', 'Unit | Route | index', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+test('it exists', function(assert) {
+  let route = this.subject();
+  assert.ok(route);
+});
```
Comment (martensonbj) [inline tests/unit/routes/index-test.js]: Assuming you delete the index.hbs file lets get rid of this file too

**Comment by lrknaff** [comment]
@martensonbj - thanks for the feedback!

Those changes have been implemented.

**Comment by nfosterky** [comment]
In the future, make sure your test name reflects what you are testing... 
(i.e. 
test('viewing the homepage'... and 
test('viewing the homepage will reroute to /reminders'... look like they are doing the same thing)

**Pull request merged**